group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -91,6 +91,27 @@ public class GroupCoordinatorConfig {
      */
     public final int genericGroupMaxSessionTimeoutMs;
 
+    /**
+     * Frequency at which to check for expired offsets.
+     */
+    public final long offsetsRetentionCheckIntervalMs;
+
+    /**
+     * For subscribed consumers, committed offset of a specific partition will be expired and discarded when
```

Comment:
nit: can we put a colon after the when and capitalize the T in this for the bullet points?